clients/appointments/src/components/index.tsx

```diff
@@ -59,6 +60,13 @@ export const AppointmentsView = ({
     setLoading(false)
   }
 
+  useEffect(() => {
+    if (!loading && !initialized) {
+      setInitialized(true)
+      callbacks?.onLoad?.((new Date().getTime() - loadStartTime.getTime()) / 1000)
```

Comment:
We will want to remove the `/1000` since we want the data in milliseconds

Reply:
I realized this once I started updating the callbacks in Jade 😅

Reply:
You could just `* 1000` in Jade. 🤪 (joking)